src/NzbDrone.Common.Test/InstrumentationTests/CleanseLogMessageFixture.cs

```diff
@@ -54,16 +54,24 @@ public class CleanseLogMessageFixture
         [TestCase(@"getTorrents(""mySecret"", [asdfasdf], 100, 0)")]
         [TestCase(@"""DownloadURL"":""https:\/\/broadcasthe.net\/torrents.php?action=download&id=123&authkey=mySecret&torrent_pass=mySecret""")]
 
-        // Spotify Refresh
-        [TestCase(@"https://auth.servarr.com/renew?refresh_token=mySecret")]
-
         // Plex
         [TestCase(@" http://localhost:32400/library/metadata/12345/refresh?X-Plex-Client-Identifier=1234530f-422f-4aac-b6b3-01233210aaaa&X-Plex-Product=Sonarr&X-Plex-Platform=Windows&X-Plex-Platform-Version=7&X-Plex-Device-Name=Sonarr&X-Plex-Version=3.0.3.833&X-Plex-Token=mySecret")]
```

Comment:
Don't need Plex test, can you add a goodreads test though please.

Reply:
good reads exists; see further down in the fixture